.cirrus.yml

```diff
@@ -5,6 +5,7 @@ task:
   gke_container:
     dockerfile: "ci/docker/build/Dockerfile"
     builder_image_name: docker-builder # gce vm image
+    builder_image_project: flutter-cirrus
```

Comment:
what is this? is this the gcp project that owns the storage bucket we upload images to? or the project that owns the `docker-builder` vm?

Reply:
This is the project the docker image is uploaded/downloaded from.

Reply:
cool, i found the project